### ISSUE TITLE
Update TF-WorldEdit

### ISSUE THREAD

**Issue by vj13573** [TotalFreedom/TotalFreedomMod#894]
See this thread.
http://totalfreedom.boards.net/thread/30958/update-worldedit

When will it be updated??? The suggestion was made months ago, that's why im bumping it because this 2 month old suggestion hasn't been resolved.

You might ask what's going on here, basically TF-WorldEdit is bugged, the clue is in this log extract:
[06:19:14] [Server thread/WARN]: [WorldEdit]
**********************************************
** This WorldEdit version does not fully support your version of Bukkit.
**
** When working with blocks or undoing, chests will be empty, signs
** will be blank, and so on. There will be no support for entity
** and biome-related functions.
**
** Please see http://wiki.sk89q.com/wiki/WorldEdit/Bukkit_adapters
**********************************************

Basically pasting signs from schematics or other containers etc are blank/empty, and that isn't suppose to happen. So please update or re-compile TF-WorldEdit so it works again.

Thanks.

**Comment by JeromSar** [comment]
As per http://totalfreedom.boards.net/thread/11906/important-first-plugin-suggestions-reports,
I do not check the forums for suggestions.

Regarding this issue, the plugin will most likely be updated when TFM 5.0 is released.

**Comment by vj13573** [comment]
Ok, I pasted the thread here:

"Hi,

It's come to my attention again for another one of these suggestions. WorldEdit needs to be updated from the Bukkit website:
dev.bukkit.org/bukkit-plugins/worldedit

Here's some issues why, the current worldedit plugin on the server has bugs:
1. Failing to load words on signs when pasting schematics or pasting anyway.
2. Failing to include items in containers or chests or etc.
3. Wayy more...

Can we PLEASE update it? I know it may be very minor to swap for another version of worldedit just because of small things but some builds that may have a lot of signs need them and we can't have be wasting so much time 24/7 everytime we paste editing signs or filling containers that we want. 
Because im pretty sure there has been a suggestion before about this issue, which needs to be resolved.

And just a final note: It will work, my current worldedit on my server is able to paste these details, therefore so can here. We need to get rid of this broken worldedit plugin and replace with a better one.

Read more: http://totalfreedom.boards.net/thread/30958/update-worldedit?page=1#ixzz3r14bjKO9"

**Comment by OxLemonxO** [comment]
@vj13573 The worldedit on the server just needs to be updated, The repo is already updated.

**Comment by Wild1145** [comment]
Please take this issue to the TF-Github repo as it has nothing to do with this project.

**Comment by JeromSar** [comment]
This issue is unrelated in TFM.
